apps/docs/content/getting-started/individual-packages.mdx

```diff
@@ -2,32 +2,82 @@
 title: Individual Packages
 description: An how to guide to install Hopper UI packages individually.
 order: 3
+status: ready
 ---
 
 If you need a particluar installation, you can install Hopper UI packages individually.
 
-## Components
+<FeatureFlag flag="alpha">
+    ## Components
 
-Hopper Components are React Components, if you are using another framework, or React Native, you should use [components tokens](/tokens/getting-started/introduction) directly when developing your own components.
+    Hopper Components are React Components, if you are using another framework, or React Native, you should use [components tokens](/tokens/overview/introduction) directly when developing your own components.
+
+    ### Installation
+
+    Run one of the following commands in your terminal to install the Hopper Components package.
+
+    <PackageInstallation library="components" mod="prod" />
```

Comment:
Some packages are missing for the components to work: `@hopper-ui/styled-system react-aria react-aria-components` they are peer-dependent and must be installed by the customer for them to work